src/cirrus/management/deployment.py

```diff
@@ -351,3 +356,108 @@ def fetch_payload(self, payload_id: str, rerun: bool | None = None):
                         payload_id,
                         e,
                     )
+
+    def get_workflows(self) -> dict[str, Any]:
```

Comment:
We use parameter store as a source for CLI configuration... I'm wondering if porting this literally is really a good idea. Would it be better to have a parameter store prefix for workflows? The names could be the "workflow ID" and the value could be the configuration?

I don't want to hold this PR up just for this, but I am also a bit wary of doing any thing that's going to further couple us to the catalog.json abomination. We should probably discuss. Perhaps there's a middle ground where we omit this function for now?

Reply:
Seconded.

Reply:
Convinced. Removed as per middle ground.